### DISCUSSION TITLE
第5节课笔记(17班)

### DISCUSSION THREAD

**Discussion by Jianfeng777** [InternLM/Tutorial#301] [笔记]
请将第五节课作业上传到这里哦！

**Comment by StarDuster** [comment]
https://gist.github.com/StarDuster/3301b7e9805e738b9f084011eff69eea

**Comment by fine-lab** [comment]
https://blog.csdn.net/SmallZCan_t/article/details/135597065?spm=1001.2014.3001.5501

**Comment by l241025097** [comment]
第五节课笔记
![05LMDeploy大模型量化部署实践读书笔记](https://github.com/InternLM/tutorial/assets/31737569/c5438b59-6def-4a31-b441-929ce15723e2)

**Comment by futtttttttttttt** [comment]
https://zhuanlan.zhihu.com/p/678770743

**Comment by AxrLH** [comment]
笔记：
https://blog.csdn.net/Axr_Samuel/article/details/135661888?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135661888%22%2C%22source%22%3A%22Axr_Samuel%22%7D

**Comment by zcg-joker** [comment]
[第五节课笔记](https://blog.csdn.net/m0_55764641/article/details/135720626)
